framework/helpers/BaseHtml.php

```diff
@@ -1693,11 +1693,18 @@ public static function renderSelectOptions($selection, $items, &$tagOptions = []
         $encodeSpaces = ArrayHelper::remove($tagOptions, 'encodeSpaces', false);
         $encode = ArrayHelper::remove($tagOptions, 'encode', true);
         if (isset($tagOptions['prompt'])) {
-            $prompt = $encode ? static::encode($tagOptions['prompt']) : $tagOptions['prompt'];
+            $attrs = ['value' => ''];
```

Comment:
@SilverFire Me too. I tried to keep the same style that already used in this method above. Is it OK to use full name only in this place?